src/components/input/Input.tsx

```diff
@@ -143,7 +144,7 @@ class Input extends Component<Props, State> {
             this.focusedMode();
           }}
           onBlur={(e) => {
-            let newNumber = parseFloat(this.state.editedValue);
+            let newNumber = parseFloat(parse(this.state.editedValue).evaluate().toString());            
```

Comment:
If this is parsed as numeric by mathjs we don’t need an extra toString and parseFloat here